data_steward/cleaners/clean_cdr.py

```diff
@@ -1,85 +1,17 @@
 """
 A module to serve as the entry point to the cleaners package.
 """
-# Python imports
+# python imports
 import logging
 
-# Third party imports
-import googleapiclient
-from google.appengine.api import app_identity
-import oauth2client
-
 # Project imports
 import bq_utils
-#import constants.bq_utils as bq_consts
-import constants.cleaners.combined as combined_consts
-import constants.cleaners.combined_deid as deid_consts
-import constants.cleaners.ehr as ehr_consts
-import constants.cleaners.rdr as rdr_consts
-import constants.cleaners.unioned as unioned_consts
-
-LOGGER = logging.getLogger(__name__)
-
-def _add_console_logging(add_handler):
-    # this config should be done in a separate module, but that can wait
-    # until later.  Useful for debugging.
-    logging.basicConfig(level=logging.DEBUG,
-                        filename='/tmp/cleaner.log',
-                        filemode='a')
-
-    if add_handler:
-        handler = logging.StreamHandler()
-        handler.setLevel(logging.INFO)
-        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-        handler.setFormatter(formatter)
-        LOGGER.addHandler(handler)
-
-
-def _clean_dataset(project=None, dataset=None, statements=None):
```

Comment:
I really don't like this being made a public function in its own module.  I don't want anyone to call this outside of the restrictions we place on it because I think clean rules should be run as a group.  That's my opinion and it is open for discussion.